contracts/scripts/utils.js

```diff
@@ -31,110 +31,93 @@ async function main() {
   console.log('Token address:', token.address);
 }
 
-async function cancelTransaction(nonce) {
+// Helpers
+
+/**
+ * @param number nonce
+ * @param string gasPrice
+ * If a transaction is stuck, you can use this function to cancel that transaction as long as the gasPrice value is larger
+ */
+const cancelTransaction = async (nonce, gasPrice) => {
   const tx = await deployer.sendTransaction({
     to: config[NETWORK].CONTRACT_ADDRESS,
     value: 0,
     nonce: nonce,
-    gasPrice: ethers.utils.parseUnits('170', 'gwei'),
+    gasPrice: ethers.utils.parseUnits(gasPrice, 'gwei'),
     gasLimit: 100000,
   });
 
   console.log(tx);
-  // await tx.wait();
   console.log('done!');
-}
-
-async function createCohort(cohortId, limit, merkleRoot) {
-  const tx = await contract.createCohort(cohortId, limit, merkleRoot, {
-    gasPrice: ethers.utils.parseUnits('100', 'gwei'),
-  });
-  console.log(tx);
-  await tx.wait();
-}
+};
 
-async function fetchCohort(cohortId) {
+/**
+ * @param string cohortId
+ * Fetches the cohort on the contract and prints out the Limit number of the cohort
+ */
+const fetchCohort = async (cohortId) => {
   const tx = await contract.cohorts(cohortId);
   console.log(tx);
   console.log('LIMIT:', tx[0].toNumber());
```

Comment:
do we need these

Reply:
Yes essentially these funcs are for printing out info like this